nupic/research/frameworks/vernon/mixins/oml.py

```diff
@@ -123,21 +155,53 @@ def create_test_test_dataloader(cls, config, dataset):
             pin_memory=torch.cuda.is_available(),
         )
 
+    def pre_task(self, tasks):
+        """Re-initialize task params prior to meta-train training."""
+        super().pre_task(tasks)
+        if not self.reset_task_params:
```

Comment:
Suggestion to to do `if self.reset_task_params: <code>`. There is no need for the return here since this function isn't expected to return anything.